lib/Parser/rterrors.h

```diff
@@ -366,6 +366,9 @@ RT_ERROR_MSG(JSERR_CantRedefineProp, 5667, "Cannot redefine property '%s'", "Can
 RT_ERROR_MSG(JSERR_FunctionArgument_NeedArrayLike, 5668, "%s: argument is not an array or array-like object", "Array or array-like object expected", kjstTypeError, 0)
 RT_ERROR_MSG(JSERR_FatalMemoryExhaustion, 5669, "", "Encountered a non-recoverable OOM", kjstError, 0)
 
+// Built In functions Errors
+RT_ERROR_MSG(JSERR_BuiltInNotAvailable, 5670, "", "Built In functions are not available.", kjstTypeError, 0)
+
```

Comment:
Good idea !

---
In reply to: [115377532](https://github.com/Microsoft/ChakraCore/pull/2898#discussion_r115377532) [](ancestors = 115377532)